tools/search_files.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+# Copyright 2008 the V8 project authors.
+# Copyright 2023 Microsoft Inc.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
```

Comment:
What LICENSE file this is referring to? Do we need to check with the foundation about this?